online-server/pom.xml

```diff
@@ -38,18 +35,16 @@
             <groupId>com.google.code.gson</groupId>
```

Comment:
It's Ok for the existing modules which will be managed by the open issue. Ensure that the new online Rest module does not depend on gson